tests/unit/models/cart-line-item-model-test.js

```diff
@@ -124,8 +123,8 @@ test('it proxies values in attrs that we would like to expose', function (assert
   assert.equal(model.id, model.attrs[GUID_KEY]);
   assert.equal(model.variant_id, model.attrs.variant_id);
   assert.equal(model.product_id, model.attrs.product_id);
-  assert.equal(model.image, model.attrs.image);
-  assert.equal(model.imageVariants, model.attrs.image_variants);
+  assert.equal(model.image.id, model.attrs.image.id);
+  assert.equal(model.imageVariants.length, 10);
```

Comment:
not without exporting the `variants` array from image-model.js. Which i guess i could.